nodejs/week1/src/backend/mealsWithReviews.js

```diff
@@ -0,0 +1,16 @@
+const meals = require("./data/meals");
+const reviews = require("./data/reviews");
+
+const resultArr = [];
+meals.forEach((meal) => {
+  const newMeal = meal;
+  newMeal["reviews"] = [];
```

Comment:
Why did you introduce newMeal? Would it make any difference if you would have written meal["reviews"]=[] instead of newMeal["reviews"]?

Reply:
@jslambda  
I thought  it will change the meal array. But now I understand that I import it, and can change it and I dont't need extra array.

Reply:
@jslambda Hi. Is it now correct?

Reply:
The statement newMeal=meal simply copies the reference to meal to newMeal. It does not allocate a new array.